static/js/components/CandidateList.jsx

```diff
@@ -623,14 +623,41 @@ const CandidateList = () => {
               label="NOT SAVED"
               className={classes.itemPaddingBottom}
             />
-            <br />
-            <div className={classes.saveCandidateButton}>
-              <SaveCandidateButton
-                candidate={candidateObj}
-                userGroups={userAccessibleGroups}
-                filterGroups={filterGroups}
-              />
-            </div>
+          </div>
+        )}
+        {Boolean(
```

Comment:
Is this Boolean in the right place?  Why is it here?  Shouldn't there be a check on the size of the filtered response?

This whole check is a bit opaque to me; perhaps worth factoring out and giving it a name?

Reply:
Yes, this is in the right place. When I omitted `Boolean` and just left the expression, a `0` was sometimes rendered when the length was zero.

I can add in-line comments to clarify what is being checked here.

Reply:
Done